app/views/receipt/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Receipt</h1>
+
+<div>Item: <%= @name %></div>
```

Comment:
This looks like you're rendering the name of the receipt as the name of the item.  I think for the second iteration if you try having a receipt with 3 items on it, you'll be able to see why that would be a problem.